src/components/ReportTransaction.js

```diff
@@ -78,4 +117,10 @@ class ReportTransaction extends Component {
 ReportTransaction.displayName = 'ReportTransaction';
 ReportTransaction.defaultProps = defaultProps;
 ReportTransaction.propTypes = propTypes;
-export default ReportTransaction;
+export default compose(
```

Comment:
NAB; you don't need to wrap with `compose` if you only have 1 hoc

Reply:
+1, this should be removed